db/migrate/20220718133934_create_emails.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :emails do |t|
       t.string :email_type
       t.string :subject
-      t.string :body
+      t.text :body
```

Comment:
You can't change an existing migration. This won't change anything for people who already have the DB. You need to create a new migration where you alter the column type...